netebpfext/net_ebpf_ext_bind.c

```diff
@@ -19,6 +19,8 @@ typedef struct _bind_context_header
 // WFP filter related globals for bind hook.
 //
 
+static HANDLE _net_ebpf_extension_bind_wfp_engine_handle = NULL;
```

Comment:
Consider storing the engine handle in `net_ebpf_extension_wfp_filter_context_t` and opening it inside `net_ebpf_extension_wfp_filter_context_create`.